common/util.go

```diff
@@ -410,3 +411,45 @@ func ReadAtMost(r io.Reader, n int) ([]byte, error) {
 	}
 	return b, err
 }
+
+func enumerateNvidiaDevices() []string {
+	gpu, err := ghw.GPU()
+	if err != nil {
+		glog.Errorf("Error getting GPU info: %v", err)
+		return nil
+	}
+
+	nvidiaCardCount := 0
+	for _, card := range gpu.GraphicsCards {
+		if strings.EqualFold(card.DeviceInfo.Vendor.Name[:6], "nvidia") {
+			nvidiaCardCount += 1
+		}
+	}
+	if nvidiaCardCount == 0 {
+		glog.Warning("No nvidia devices found")
+		return nil
+	}
+
+	devices := []string{}
+
+	for i := 0; i < nvidiaCardCount; i++ {
+		s := strconv.Itoa(i)
+		devices = append(devices, s)
+	}
+
+	return devices
+}
+
+func ParseNvidiaDevicesFlag(nvidia string) []string {
```

Comment:
I wonder if there's a better place for this other than `common` as it's not really shared across packages, it's only used in `cmd`.  Don't really have a suggestion atm.

Reply:
The other obvious option is moving this in a subpackage in cmd like `cmd/utils` - but both `livepeer.go` and `livepeer_bench.go` anyway use helper functions from `common/util.go` - eg. [parsing video profiles](https://github.com/livepeer/go-livepeer/blob/3908252d23edd62590c61b02598c5fffc148014b/cmd/livepeer_bench/livepeer_bench.go#L214) for the transcoder. But I don't have a strong opinion either way.

Reply:
let's keep it in `common` then 👍